runtime/experiments/workload_mix_realworld/run.sh

```diff
@@ -105,7 +106,7 @@ run_experiments() {
 	local -i batch_id=0
 	local -i roll=0
 	local -ir total_iterations=10000
-	local -ir worker_max=30
+	local -ir worker_max=50
```

Comment:
I'm guessing these can be overridden by env variables?

Reply:
Not the way it's written currently, but something like `local -ir worker_max=${WORKER_MAX:-50}` would do with trick with a default value if it's not defined. Likely a better approach than noise diffs!